components/core/src/clp/ffi/ir_stream/UnstructuredIrDeserializerImpl.cpp

```
@@ -0,0 +1,300 @@
+ * @param encoded_vars
+ * @return A result containing the encoded text AST on success, or an error code indicating the
+ * failure:
+ * - Forwards `EncodedTextAst::create`'s return values on failure.
+ */
+[[nodiscard]] auto build_encoded_text_ast(
+        std::string const& logtype,
+        std::vector<std::string> const& dict_vars,
+        std::vector<encoded_variable_t> encoded_vars
+) -> ystdlib::error_handling::Result<EncodedTextAst<encoded_variable_t>> {
+    StringBlob string_blob;
+    for (auto const& dict_var : dict_vars) {
```

Cppcheck (2.20.0)

[style] The function 'get_auto_gen_keys_schema_tree' is never used.

(unusedFunction)